test/web-platform-tests/to-run.yaml

```diff
@@ -1107,7 +1107,6 @@ formaction.html: [fail, Unknown]
 
 DIR: html/semantics/forms/constraints
 
-form-validation-validity-patternMismatch.html: [fail, Unknown]
```

Comment:
Use `fail-node18` here (and below) to signal the right support matrix, and the tests should pass, I think!

Reply:
Thanks for the tip! I added it, but there were more tests that fail. Essentially, any test where the input has a pattern will fail. Should I add more `fail-node18` to those tests?

Reply:
Oh, I see. Well, we don't want to make jsdom (or the pattern parts of it) completely unusable on Node 18, at least until it goes out of LTS support. So I guess we need a fallback, that try/catches with "v" and then falls back to "u".